src/services/index.js

```diff
@@ -1,6 +1,5 @@
-export const CONTROLLER_URL = __DEV__
-  ? 'https://dev-logistics-wizard.mybluemix.net/api/v1'
-  : 'https://logistics-wizard.mybluemix.net/api/v1';
+export const CONTROLLER_URL = process.env.CONTROLLER_SERVICE ?
+  `${process.env.CONTROLLER_SERVICE}/api/v1` : 'https://notset/api/v1';
```

Comment:
Actually the `' || 'https://dev-logistics-wizard.mybluemix.net'` part could also be put in the config, that way it's all managed there.